cmd/kube-scheduler/app/server.go

```diff
@@ -237,10 +237,13 @@ func Run(ctx context.Context, cc *schedulerserverconfig.CompletedConfig, sched *
 	}
 
 	// Start up the healthz server.
+	serverShutdownCh := make(<-chan struct{})
+	listenerStoppedCh := make(<-chan struct{})
```

Comment:
You don't need to initialize the channels here
```suggestion
	var serverShutdownCh = <-chan struct{}
	var listenerStoppedCh = <-chan struct{}
```

Reply:
I addressed your advice by using the defer statement.
[use defer](https://github.com/kubernetes/kubernetes/pull/128888/commits/9cc325e79d67c64132431b93816a4b5d92c4a7d3)